lib/src/discovery.dart

```diff
@@ -113,10 +113,10 @@ Future<PackageConfig?> findPackageConfigUri(
 /// a best-effort attempt is made to return a package configuration.
 /// This may be the empty package configuration.
 ///
-/// If [minVersion] is greated than 1, `.packages` files are ignored.
+/// If [minVersion] is greater than 1, `.packages` files are ignored.
 /// If [minVersion] is greater than the version read from the
 /// `package_config.json` file, it too is ignored.
-Future<PackageConfig?> findPackagConfigInDirectory(Directory directory,
+Future<PackageConfig?> findPackageConfigInDirectory(Directory directory,
```

Comment:
[nit] Consider making this private since it is not referenced outside this library.